cdap-app-templates/cdap-etl/hydrator-spark-core-base/src/main/java/io/cdap/cdap/etl/spark/batch/RDDCollection.java

```diff
@@ -91,7 +92,10 @@ public RDDCollection(JavaSparkExecutionContext sec, JavaSparkContext jsc,
   public SparkCollection<T> cache() {
     SparkConf sparkConf = jsc.getConf();
     if (sparkConf.getBoolean(Constants.SPARK_PIPELINE_AUTOCACHE_ENABLE_FLAG, true)) {
-      return wrap(rdd.cache());
+      String cacheStorageLevelString = sparkConf.get(Constants.SPARK_PIPELINE_CACHING_STORAGE_LEVEL, 
+          Constants.DEFAUL_CACHING_STORAGE_LEVEL);
```

Comment:
nit: use indentation consistent with rest of project. In this situation, should line up with the argument from the previous line.